src/components/PickerYear.vue

```diff
@@ -40,6 +40,12 @@ export default {
   mixins: [
     pickerMixin,
   ],
+  props: {
+    yearPickerRange: {
```

Comment:
The prop **yearPickerRange** feels a bit too obvious, because we it's already inside the PickerYear component.

Suggestions: maybe change to something which beter fits its context, so '**amount-per-page**', 'range' or 'year-range'?

Reply:
You are right.
I changed it to `year-range`. With `year-range` you can still see whats going on with the prop and don't think `range` is some kind of feature to define a `date-range`